firefox-ios/firefox-ios-tests/Tests/XCUITests/BrowsingPDFTests.swift

```diff
@@ -37,6 +37,10 @@ class BrowsingPDFTests: BaseTestCase {
         // Click on a link on the pdf and check that the website is shown
         app.links.element(boundBy: 0).tapOnApp()
         waitUntilPageLoad()
+        let checkboxValidation = app.webViews["Web content"].staticTexts["Verify you are human"]
```

Comment:
Let's try this until we can create our own pdf site with a link cc @mdotb-moz

Reply:
yea, it should be temporary